docs/command_reference.md

```diff
@@ -294,4 +294,4 @@ If you are receiving this error:
 * fatal: Get <URL>/v2/: x509: certificate signed by unknown authority
 ```
 The Notary CLI must be configured to trust the root certificate authority of the server it is communicating with.
-This can be configured by specifying the root certificate with the `--tlscacert` flag or by editing the Notary client configuration file.  Additionally, you can add the root certificate to your system CAs.
\ No newline at end of file
+This can be configured by specifying the root certificate with the `--tlscacert` flag or by editing the Notary client configuration file.  Additionally, you can add the root certificate to your system CAs.
```

Comment:
why is there a diff here?

Reply:
@justincormack it usually means a trailing newline character was added/removed.

Reply:
Yes but there doesn't appear to be one in this case thats why I asked...

Reply:
Would you like me to rebase?

Reply:
That is strange!

Reply:
@justincormack Looks like a GitHub UI bug; if you look at the raw patch (https://github.com/theupdateframework/notary/pull/1552.patch), it _does_ show the "missing newline at end of file" before the PR

```patch
 The Notary CLI must be configured to trust the root certificate authority of the server it is communicating with.
-This can be configured by specifying the root certificate with the `--tlscacert` flag or by editing the Notary client configuration file.  Additionally, you can add the root certificate to your system CAs.
\ No newline at end of file
+This can be configured by specifying the root certificate with the `--tlscacert` flag or by editing the Notary client configuration file.  Additionally, you can add the root certificate to your system CAs.
```

Reply:
I've rebased this branch. It looks like the diff is now resolved.